allies.MD

```diff
@@ -0,0 +1,19 @@
+# Resources for allies
+
+## Why does it matter to you?
+Put shortly:
+- If you advertise jobs - [the wording you use impacts who applies](https://www.glassdoor.com/employers/blog/10-ways-remove-gender-bias-job-listings/) (great candidates might be self-excluding).
+- If you hire candidates - [the techniques you use impact who gets through](https://hbr.org/2017/06/7-practical-ways-to-reduce-bias-in-your-hiring-process) (you might not hire or promote the best candidates).
+- If you intend to retain employees - [the culture of your organisation impacts who wants to stay](https://leanin.org/women-in-the-workplace-report-2022/were-in-the-midst-of-a-great-breakup) (including genuine inclusion of diversity and equitable opportunities).
+
+Action on each of these matters is completely up to you, businesses are entitled to hire whoever and however they want!
+But when systems aren't working in the businesses' interest, there's certainly more to it than altruism.
+
+## Outsource!
+It can be hard to know where to start, but you're not the first in wanting to implment a Diversity, Equity and Inclusion program.
+
+Trainers such as LifeLabs run sessions to highlight ways de-biasing improves your interview process (and how to do it at your company).
+Get a program designed for your company [here](https://home.lifelabslearning.com/dei-programs), or attend an event for the public such as [this one](https://lifelabslearning.com/course/inclusive-interviewing-december-8-10-12pm-et/).
```

Comment:
Please, consider removing this section, or fitting it with a free initiative according to the initiatives shown in this repository.